### PR TITLE
Bugfix/init value for pointers

### PR DESCRIPTION
**Pull request by mgallien**

### CODE REVIEW

File: src/gui/accountsettings.h [hunk 1]
```diff
@@ -143,8 +143,8 @@ private slots:
     bool _wasDisabledBefore;
     AccountState *_accountState;
```
Comment (allexzander) [inline src/gui/accountsettings.h:144]: @mgallien this one should also be set to `nullptr`?

Comment (mgallien) [reply src/gui/accountsettings.h:144]: the reason is that the constructor is always putting a value in it so a default value here would be always overriden

**Review by allexzander — APPROVED** [review]

**Comment by claucambra** [comment]
Ugh

**Pull request merged**